app/controllers/two_factor_authentication/webauthn_verification_controller.rb

```diff
@@ -40,10 +40,17 @@ def handle_webauthn_result(result)
     end
 
     def handle_valid_webauthn
-      handle_valid_verification_for_authentication_context(auth_method: 'webauthn')
+      if form.webauthn_configuration.platform_authenticator
+        handle_valid_verification_for_authentication_context(
+          auth_method: TwoFactorAuthenticatable::AuthMethod::WEBAUTHN,
+        )
+      else
+        handle_valid_verification_for_authentication_context(
+          auth_method: TwoFactorAuthenticatable::AuthMethod::WEBAUTHN_PLATFORM,
+        )
+      end
```

Comment:
yeah, added in ebaed16605

Reply:
Actually, it got kind of weird since we don't want to use params in the authentication call so I backed it out, but I did replace the `'webauthn'` and `'webauthn_platform'` with the respective constants below.